src/ORM/Table.php

```diff
@@ -1226,6 +1208,35 @@ public function get($primaryKey, $options = [])
     }
 
     /**
+     * @param int|string|array $primaryKey Primary key value.
+     *
+     * @return array Conditions.
+     */
+    public function primaryKeyToConditions($primaryKey)
```

Comment:
Does this method _need_ to be public? If we don't need to add more API surface area I'd like to avoid it.